src/polymer-project.ts

```diff
@@ -181,17 +175,25 @@ export class PolymerProject {
     });
   }
 
-  // TODO(justinfagnani): add options, pass to vfs.src()
   dependencies(): NodeJS.ReadableStream {
-    let deps = this.dependencyGlobs;
-    return vfs.src(deps, {
-      allowEmpty: true,
-      cwdbase: true,
-      nodir: true,
-    });
-  }
+    let dependenciesStream: NodeJS.ReadableStream = forkStream(
```

Comment:
I built this to purposefully match `.sources()`, which currently creates a brand new readable streams for the consumer. 

This sounds like a somewhat useful feature (you could create additional streams for additional build tasks without worrying about how data is modified) but you could do the same with fork() on an as needed basis.

I'm good with either as long as both `sources()` & `dependencies()` behave the same. I'll keep it as is since I don't see a good reason for/against but let me know if I'm missing something.